### ISSUE TITLE
[DropDownMenu][Select] Add title and description for touch screen menu

### ISSUE THREAD

**Issue by ramtinabadi** [Vieolo/vieolo-ui#317]
In the touch screen menu, a menu comes up from the bottom of the screen.

In the desktop layout, it is quite obvious where the source of the dropdown is. But in the touch screen, when a modal like menu is opened, it can be confusing for the user as they may not be sure where the source was.

So, there should be the ability to display a title and description on the opened modal.
